vehicle/DieselEngine.h

```diff
@@ -5,18 +5,18 @@
 
 class DieselEngine final : public IEngine {
 public:
-    DieselEngine(int cylinders) : cylinders(cylinders) {}
+    DieselEngine(int m_cylinders) : m_cylinders(m_cylinders) {}
```

Comment:
same here

Reply:
updated, p_ for parameters, m_ for class members